auth-foundation/src/main/java/com/okta/authfoundation/client/OidcConfiguration.kt

```diff
@@ -118,10 +128,12 @@ class OidcConfiguration @InternalAuthFoundationApi constructor(
     }
 
     /** The Json object to do the decoding from the okta server responses. */
-    @InternalAuthFoundationApi val json: Json = defaultJson()
+    @Transient @InternalAuthFoundationApi val json: Json = defaultJson()
 
-    internal companion object {
+    companion object {
         internal fun defaultJson(): Json = Json { ignoreUnknownKeys = true }
+
+        lateinit var default: OidcConfiguration
```

Comment:
What is `OidcConfiguration.default` used for / where does its configuration come from?

Reply:
Is this the equivalent to `okta-mobile-swift`'s Okta.plist shared configuration?

Reply:
That's correct. It's the same idea as okta-mobile-swift's plist configuration.